Gemfile.lock

```diff
@@ -170,7 +170,7 @@ GEM
     holidays (2.2.0)
     htmlentities (4.3.4)
     i18n (0.7.0)
-    icalendar (1.4.3)
+    icalendar (2.3.0)
```

Comment:
There's a compatibility warning install message but it looks like we're only using `#parse` which doesn't appear to be affected